src/docfx/build/page/PageBuilder.cs

```diff
@@ -380,7 +375,6 @@ private void ProcessConceptualHtml(ErrorBuilder errors, FilePath file, string ht
         _searchIndexBuilder.SetBody(file, searchText.ToString());
 
         conceptual.Conceptual = LocalizationUtility.AddLeftToRightMarker(_buildOptions.Culture, result);
-        conceptual.WordCount = wordCount;
     }
```

Comment:
Should we remove this line too as I see we are adding it in `TemplateEngine`

Reply:
I think there is no difference to put `AddLeftToRightMarker` here or later in applying template.
Prefer to put it here to keep the behavior less changed.

Reply:
Had an offline discussion with @yufeih, the output before applying temple should be more complete, so we should keep this logic here.